atlasdb-jepsen-tests/Dockerfile

```diff
@@ -1,5 +1,3 @@
 FROM tjake/jepsen:latest
 
-ADD . /jepsen/atlasdb
```

Comment:
If you're removing this, you can just override the command directly in the docker-compose file and not have this file at all.